src/java/htsjdk/samtools/CRAMFileReader.java

```diff
@@ -78,15 +78,8 @@ public CRAMFileReader(final File cramFile, final InputStream inputStream) {
      *                        reference sequences
      */
     public CRAMFileReader(final File cramFile, final InputStream inputStream,
-                          final ReferenceSource referenceSource) {
-        if (cramFile == null && inputStream == null)
-            throw new IllegalArgumentException(
-                    "Either file or input stream is required.");
-
-        this.cramFile = cramFile;
-        this.inputStream = inputStream;
-        this.referenceSource = referenceSource;
-        getIterator();
+                          final ReferenceSource referenceSource) throws FileNotFoundException, IOException {        
```

Comment:
same style issue "it's bad style to have 2 exceptions declared and one of them be a subclass of the other: `FileNotFoundException extends IOException`"